python/langsmith/client.py

```diff
@@ -122,6 +128,36 @@ def validate_api_key_if_hosted(cls, values: Dict[str, Any]) -> Dict[str, Any]:
                 )
         return values
 
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        api_url: Optional[str] = None,
+        retry_config: Optional[Retry] = None,
+        timeout_ms: Optional[int] = None,
+        **kwargs: Any,
+    ) -> None:
+        kwargs = kwargs or {}
+        if api_key is not None:
```

Comment:
pydantic is sad

Reply:
:( It also messes up the docstrings / type hinting when we add an `__init__` method, unfortunately. Should I just stop using pydantic for the client? I think that would be good..